.github/ISSUE_TEMPLATE/new-feature.yaml

```
@@ -28,12 +28,10 @@ body:
   - type: textarea
     id: deadline
     attributes:
-      label: Release date
+      label: Release Date
```

```suggestion
      label: Release date
```
Looking back on the templates I'd prefer to uncapitalize these topics to match our docs site style, but don't feel strongly.